frontend/start-dev.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+function clean_up() {
+  kill -15 %1
+  kill -15 %2
+  kill -15 %3
+}
+
+echo "Preparing dev env for KFP frontend"
+
+echo "Detecting api server pod names..."
+MLMD_ENVOY_POD=$(kubectl get pods -l component=metadata-envoy -o=custom-columns=:.metadata.name --no-headers)
```

Comment:
This line does not seem to be passing the namespace. Can this be a problem?

Reply:
Good catch! Thanks. I should add the namespace.

I've been using kubeflow namespace as default, so I overlooked this one.
BTW, these changes are from a separate PR: https://github.com/kubeflow/pipelines/pull/2104